### PR TITLE
[fix][meta] Bump jetcd to 0.7.7

### PR DESCRIPTION
**Pull request by liangyuanpeng**
Fixes #xyz

Main Issue: #xyz

PIP: #xyz 

### Motivation

Bump jetcd to 0.7.7, it's major fix two bug and Upgrade of related dependencies.

Bug fixes:

- https://github.com/etcd-io/jetcd/pull/1240
- https://github.com/etcd-io/jetcd/pull/1256

Full details: https://github.com/etcd-io/jetcd/compare/jetcd-0.7.5...jetcd-0.7.7

### Verifying this change

- [x] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [x] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] The metrics
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository: https://github.com/pulsar-sigs/pulsar/pull/6

### CODE REVIEW

**Comment by liangyuanpeng** [comment]
I review this PR again. The main two bugfixes are for 0.7.6 and pulsar uses 0.7.5, so this PR can wait a little longer.

File: distribution/server/src/assemble/LICENSE.bin.txt [hunk 1]
```diff
@@ -436,6 +436,7 @@ The Apache Software License, Version 2.0
     - io.grpc-grpc-rls-1.55.3.jar
     - io.grpc-grpc-servlet-1.55.3.jar
     - io.grpc-grpc-servlet-jakarta-1.55.3.jar
+    - io.grpc-grpc-util-1.60.0.jar
```
Comment (lhotari) [inline distribution/server/src/assemble/LICENSE.bin.txt:439]: Mixing grpc library versions doesn't look good. grpc version is currently coupled to protobuf version and also to bookkeeper version so we cannot safely upgrade protobuf. (see thread https://lists.apache.org/thread/s9g6w31vtwzgqf162hhlcr2nx3y68gv5)

File: distribution/server/src/assemble/LICENSE.bin.txt [hunk 2]
```diff
@@ -478,6 +480,7 @@ The Apache Software License, Version 2.0
     - io.vertx-vertx-web-4.3.8.jar
     - io.vertx-vertx-web-common-4.3.8.jar
     - io.vertx-vertx-grpc-4.3.5.jar
+    - io.vertx-vertx-grpc-4.5.1.jar
```
Comment (lhotari) [inline distribution/server/src/assemble/LICENSE.bin.txt:483]: Mixing vertx libraries versions doesn't look good.

**Review by lhotari — CHANGES_REQUESTED** [review]
Added review comments about mixing grpc and vertx library versions.

**Pull request closed without merging**